AfricanReligions/localisation/afr_event_texts.csv

```diff
@@ -78,4 +78,28 @@ EVTDESC_MNM_3227_AFRICAN;As I step up to the [Root.GetReligiousCeremonyLocation]
 EVTDESC_MNM_3229_BORI;I walk to the center of the ceremony and make a quick decision � as I reach a group of people, I collapse to the floor. The worshipers gasp as I, their [Root.GetTitle], begin singing [Root.Religion.GetScriptureName]. I roll my eyes back as I sing, as if confronting the skies themselves. When my last words ring out, they start to whisper. For a moment I am convinced that my stunt has succeeded. Then the [Root.Religion.GetPriestTitle] prods at me. "Uhm, my [Root.GetLordLady]? Let's get you back onto your feet.";;;;;;;;;;;;;x
-EVTDESC_MNM_3229_AFRICAN;I step forwards and make a bold decision � as I reach a group of people, I collapse to the floor. The worshipers gasp as I, their [Root.GetTitle], begin quoting fragments of [Root.Religion.GetScriptureName]. I roll my eyes back as I speak, as if confronting the skies themselves. When my last words ring out, they start to whisper. For a moment I am convinced that my stunt has succeeded. Then the [Root.Religion.GetPriestTitle] prods at me. "Uhm, my [Root.GetLordLady]? Let's get you back onto your feet.";;;;;;;;;;;;;x
\ No newline at end of file
+EVTDESC_MNM_3229_AFRICAN;I step forwards and make a bold decision � as I reach a group of people, I collapse to the floor. The worshipers gasp as I, their [Root.GetTitle], begin quoting fragments of [Root.Religion.GetScriptureName]. I roll my eyes back as I speak, as if confronting the skies themselves. When my last words ring out, they start to whisper. For a moment I am convinced that my stunt has succeeded. Then the [Root.Religion.GetPriestTitle] prods at me. "Uhm, my [Root.GetLordLady]? Let's get you back onto your feet.";;;;;;;;;;;;;x
+EVTDESC_AFRREL00500;In the ethereal realm of Jan Gari, the Bori spirits exist in a transcendental state incomprehensible to the mortals. If I make special contact with one of the great bori clans and enter into communion with their king, it is possible that the spirits will grant me their blessing.;;;;;;;;;;;;;x
+EVTOPTA_AFRREL00501;Time to chose.;;;;;;;;;;;;;x
+EVTOPTA_AFRREL00502;Maybe another time.;;;;;;;;;;;;;x
+EVTDESC_AFRREL00510;While I make an effort so that my soul comes in contact with the Jan Gari plan, I face the decision of which spirit I should ask for favor. There are six major categories of spirits each with their own hierarchy and a powerful spiritual king who rules over them. Which of these spiritual courts will I pay homage to?;;;;;;;;;;;;;x
+EVTOPTA_AFRREL00512;Yan Riga, the city spirits.;;;;;;;;;;;;;x
+EVTOPTA_AFRREL00513;Babbaku, the rural spirits.;;;;;;;;;;;;;x
+EVTOPTA_AFRREL00514;Tan Garki, the warrior spirits.;;;;;;;;;;;;;x
+EVTOPTA_AFRREL00515;More options.;;;;;;;;;;;;;x
```

Comment:
How about something more immersive, like "There are other spirits to choose..." or "I had something else in mind."